*** cadasta.orig	2016-01-14 10:17:05.467290681 +0000
--- cadasta	2016-01-14 10:19:11.183291290 +0000
*************** server {
*** 13,18 ****
--- 13,22 ----
          proxy_no_cache $cookie_auth_tkt;
          proxy_cache_valid 30m;
          proxy_cache_key $host$scheme$proxy_host$request_uri;
+         proxy_redirect http://cadasta-vagrant.local/ http://cadasta-vagrant.local:7000/;
+         sub_filter http://cadasta-vagrant.local/ http://cadasta-vagrant.local:7000/;
+         sub_filter_once off;
+         sub_filter_types "*";
      }
  }

